commandServer/routes.py

```diff
@@ -0,0 +1,20 @@
+import json
+
+
+class BaseModel(object):
+
+    def get(self, request, params=None):
+        return json.dumps([]).encode('utf_8')
```

Comment:
It will be implemented in future in base class for view. It's just example for future Kataryna work.